.github/workflows/trivy.yml

```diff
@@ -16,40 +16,31 @@ jobs:
   build:
     strategy:
       matrix:
-        arch: [amd64, arm64]
```

Comment:
The aws-cli build takes 5-10 minutes so I decided to remove arm64 from the scanner.